Makefile

```diff
@@ -10,7 +10,7 @@ SELF:=$(firstword $(MAKEFILE_LIST))
 
 PKG=hgvs
 PKGD=$(subst .,/,${PKG})
-TEST_DIRS:=.,doc,hgvs,tests
```

Comment:
Hi Lucaswiman, the . is intended to invoke doc test of README.rst since it contains example python code showing how to use hgvs.